R/fiveyeargwsum-styles.R

```diff
@@ -1,12 +1,20 @@
-getFiveyearStyle <- function(data, info=NULL) {
+getFiveyearStyle <- function(data, info=NULL, invertedPlot = FALSE) {
 
     x <- data[[1]]$time
     y <- data[[1]]$value
+    
+    #semantics for min/max are swapped on inverted plots
+    minMaxLabels = c("Max. Instantaneous", "Min. Instantaneous");
+    if(invertedPlot) {
+      minMaxLabels = c("Min. Instantaneous", "Max. Instantaneous");
+    }
+    
```

Comment:
Actually, I can't completley use that pattern because the legend name needs to be mixed with y (measurement number), so I'll use the first pattern you posted.